src/core_plugins/timelion/public/chain.peg

```diff
@@ -91,7 +91,7 @@ function_name
  = first:[a-zA-Z]+ rest:[.a-zA-Z0-9_-]* { return first.join('') + rest.join('') }
 
 function "function"
- = space? '.' name:function_name space? '(' space? arg_list:arg_list? space? ')' {
```

Comment:
i don't really get it ... how does this solve 'hitting enter now submits the query without adding a newline' ?